src/uwtools/config.py

```diff
@@ -70,6 +73,17 @@ def __init__(self, config_path=None):
 
         self.config_path = config_path
 
+        #initialize logger with parent inheritance
+        if __name__ == '__main__':
+            _name = __name__
+        else:
+            py_caller = getmodule(stack()[1][0])
+            _name = f"{py_caller.__name__}.{__name__}"
+        self.log = Logger(name=_name).get_logger()
+        #pylint: disable=no-member, consider-using-f-string
+        message = "existing" if _name in logging.root.manager.loggerDict else "new"
+        self.log.debug("Beginning {message} logging with {name}", message=message, name=_name)
```

Comment:
Should use a test of this in test_config.py, may be an additional issue to resolve after this

Reply:
Added issue #189